_layouts/person.html

```diff
@@ -156,7 +156,9 @@ <h1 class="person-name">{{ person.title | escape_once }}</h1>
       </div>
 
       <h3 class="person-show-title">
+       <a href="{{ record.item.url }}">
```

Comment:
Yeep, and for quick layout changes ideally you'd want to be running `gulp debug` rather than `build`